recipes/r-gdalraster/meta.yaml

```diff
@@ -0,0 +1,134 @@
+{% set version = '2.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-gdalraster
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/gdalraster_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/gdalraster/gdalraster_{{ version }}.tar.gz
+  sha256: 5a52e24e10e44163dfc366007a356dfc27f9d28353d557b507e46d119f3b6ef2
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - r-rcpp
+    - r-rcppint64
+    - r-bit64
+    - r-nanoarrow
+    - r-wk
+    - r-xml2
+    - {{ compiler('c') }}              # [not win]
+    - {{ stdlib("c") }}                # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ stdlib("c") }}                # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+  host:
+    - r-base
+    - r-rcpp >=1.0.7
+    - r-rcppint64
+    - r-bit64
+    - r-nanoarrow
+    - r-wk
+    - r-xml2
+    - libgdal-core
+  run:
+    - r-base
+    - r-rcpp >=1.0.7
+    - r-rcppint64
+    - r-bit64
+    - r-nanoarrow
+    - r-wk
+    - r-xml2
+
+test:
+  commands:
+    - $R -e "library('gdalraster')"           # [not win]
+    - "\"%R%\" -e \"library('gdalraster')\""  # [win]
```

Comment:
```suggestion
test:
  requires:
    - r-testthat
  commands:
    - $R -e "library('gdalraster')"  # [not win]
    - $R -e "testthat::test_file('tests/testthat.R', stop_on_failure=TRUE)"  # [not win]
    - "\"%R%\" -e \"library('gdalraster')\""  # [win] 
    - "\"%R%\" -e \"testthat::test_file('tests/testthat.R', stop_on_failure=TRUE)\""  # [win]
```

If `gdalraster` has a R test suite it should get run here too (also anything beyond `testthat` should get added. If not, ignore this, but consider adding any existing test suites a different way.